src/NuGet.Status/Configuration/StatusConfiguration.cs

```diff
@@ -13,6 +13,8 @@ public class StatusConfiguration : Services.Configuration.Configuration
 
         public string ApplicationInsightsKey { get; set; }
 
+        public string AdminIdentities { get; set; }
```

Comment:
Consider deserializing directly as a string array, e.g.
https://github.com/NuGet/NuGetGallery/blob/d7aaa8dd8aa213f77ca61d88eee1c4c7df2ed2d9/src/NuGetGallery.Services/Configuration/AppConfiguration.cs#L39

Reply:
I did consider this, imported the `NuGet.Services.Configuration` package to get the attribute, and it was ignored by the deserialization. Rather than spend time digging into why deserialization ignored it and returned null (for workaround code), I thought it more expedient to get this up and running with my own split statement.